analysis/organize_data.R

```diff
@@ -132,18 +116,19 @@ summary(ipumsdata$marriage)
 
 ipumsdata$employment <- factor(ifelse(ipumsdata$empstat==1, "Employed",
                                     ifelse(ipumsdata$empstat==2 |
-                                             ipumsdata$empstat==3,"Unemployed", NA)),
-                             levels=c("Employed","Unemployed"))
+                                             ipumsdata$empstat==3,"Nonemployed", NA)),
+                             levels=c("Employed","Nonemployed"))
 
 table(ipumsdata$empstat, ipumsdata$employment, exclude=NULL) 
 summary(ipumsdata$employment)
 
 #Metro status 
-ipumsdata$metros <- factor(ifelse(ipumsdata$marst==1, "NotMetro",
-                                    ifelse(ipumsdata$marst==2 |
-                                             ipumsdata$marst==3 |
-                                             ipumsdata$marst==4, "Metro", NA)),
-                             levels=c("Metro","NotMetro"))
+ipumsdata$metros <- factor(ifelse(ipumsdata$metro==0, "Mixed",
+                                  ifelse(ipumsdata$metro==1, "NotMetro",
+                                         ifelse(ipumsdata$metro==2 |
+                                                  ipumsdata$metro==3 |
+                                                  ipumsdata$metro==4, "Metro", NA))),
+                             levels=c("Metro","Mixed","NotMetro"))
```

Comment:
Again, your checking code caught the problem here, but it wasn't corrected. There are two issues:

- You are using marst to code this, probably the result of a copy and paste job. 
- You are ignoring the category of "mixed" which needs its own final category.